enhancements/kube-apiserver/stability-history-of-resources.md

```diff
@@ -0,0 +1,201 @@
+---
+title: stability-history-of-resources
+authors:
+  - "@deads2k"
+reviewers:
+approvers:
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
```

Comment:
you need to pick values for these headers right?  Or should we just drop them from the template?